CHANGELOG.md

```diff
@@ -1,3 +1,431 @@
+## v0.3.0 - 2020-07-29
+
+We're happy to announce the release of Lokomotive v0.3.0 (Coast Starlight).
+
+This release packs new features and bugfixes. Some of the highlights are:
+
+* Kubernetes 1.18.6
+* AKS Kubernetes 1.16.10
+* [Component updates](#component-updates)
+
+### Changes in v0.3.0
+
+#### Kubernetes updates
+
+- Update Kubernetes to
+[v1.18.6](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md#v1186)
+([#726](https://github.com/kinvolk/lokomotive/pull/726)).
+
+#### Platform updates
+
+##### Packet
+- Update default machine type from `t1.small.x86` to `c3.small.x86`, since
+    `t1.small.x86` are EOL and no longer available in new Packet projects
+    ([#612](https://github.com/kinvolk/lokomotive/pull/612)).
+
+  If you haven't explicitly defined the controller_type and/or
+  worker_pool.node_type configuration options, upgrading to this release will
+  replace your controller and/or worker nodes with c3.small.x86 machines. To
+  avoid this, set these configuration options to the desired values.
```

Comment:
Are you sure this will happen? I think the cluster will be broken, as terraform will delete the controllers (etcd cluster is broken) and create a new ones, and therefore loose all your data.

Am I missing something?

Reply:
Yes @rata If the machine type isn't explicity defined, it will amount to a creation of new nodes entirely.
In order to mitigate that existing clusters), the call is to explicitly state in the lokocfg file.

I copy pasted the above paragraph from the `Release Notes` in your PR #612

Reply:
Oh, didn't know it comes from there, thanks!

I still think the release notes on that PR are probably wrong. I think the cluster will be destroyed and all etcd data will be lost if you haven't explicitly defined the controller types.

And for workers it will create downtime too.

Reply:
Yes @rata that's what I said will happen. It will be a new cluster.

Reply:
We should add a bold warning here, that all cluster data will be lost if user upgrades without specifying it.

Reply:
added a **WARNING** note.

Reply:
Hm, I would rephrase that, to mention, that if not done, you can lose the entire controlplane of your cluster. I'd explain why the type changed, from what to what and include a snippet which should be added if someone do not define machine type at the moment.

Reply:
The comment already mentions why the type changed and what changed. I've also added a config snippet.

```
- Update default machine type from `t1.small.x86` to `c3.small.x86`, since
    `t1.small.x86` are EOL and no longer available in new Packet projects
    ([#612](https://github.com/kinvolk/lokomotive/pull/612)).

  **WARNING**: If you haven't explicitly defined the controller_type and/or
  worker_pool.node_type configuration options, upgrading to this release will
  replace your controller and/or worker nodes with c3.small.x86 machines thereby
  loosing all your cluster data. To avoid this, set these configuration options
  to the desired values.

  Make sure that the below variables are explicity defined in your cluster
  configuration. This only applies to machine type `t1.small.x86`.

  ```hcl
  cluster "packet" {
    .
    .
    controller_type = "t1.small.x86"
    .
    .
    worker_pool "pool-name" {
      .
      node_type = "t1.small.x86"
      .
    }
  }
  ```
```